### PR TITLE
526 remove pmr mg

### PR DESCRIPTION
**Pull request by markgreenburg**
- Removes ability to request a private medical records (PMR) release
- Adds info on how to submit a 4142 in lieu of requesting PMR release through the 526 form
- Updates help text for 'which option should I choose' with 4142 instructions
- Adds a new 'next page' if user selects option indicating doctor has their medical records. This new page contains content on 4142 submission process.
- Updates confirmation page with 4142 info (only if they chose to 1. use PMR and 2. not upload PMR directly, for any of their rated disabilities)

### ----------_Base page UI:_----------
screen shot 2018-05-18 at 9 49 54 am

### ----------_New expander under 'no' option:_----------
screen shot 2018-05-18 at 9 50 23 am

### ----------_New 'Which Should I choose?' text:_----------
screen shot 2018-05-18 at 9 50 13 am

### ----------_New Page after selecting No -> Continue_ ----------
screen shot 2018-05-18 at 5 37 02 pm

### ----------_Confirmation Page:_----------
screen shot 2018-05-18 at 9 52 29 am

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -496,10 +498,17 @@ const formConfig = {
                   'ui:options': {
                     labels: {
                       yes: 'Yes',
-                      no: 'No, please get them from my doctor'
+                      no: 'No, my doctor has my medical records'
```
Comment (cehsu) [inline src/applications/disability-benefits/526EZ/config/form.js]: Should these be booleans?

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 2]
```diff
@@ -518,6 +527,11 @@ const formConfig = {
                       type: 'string',
                       'enum': ['yes', 'no']
                     },
+                    'view:privateRecords4142Notice': {
+                      type: 'object',
+                      'ui:collapsed': true,
```
Comment (cehsu) [inline src/applications/disability-benefits/526EZ/config/form.js]: Is `hideIf` not an option here?

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 3]
```diff
@@ -539,6 +553,39 @@ const formConfig = {
...
+            disabilities: {
+              items: {
+                'ui:description': authorizationToDisclose
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
```
Comment (cehsu) [inline src/applications/disability-benefits/526EZ/config/form.js]: Can we reuse the `disabilities` definition here?

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx [hunk 4]
```diff
@@ -36,8 +36,21 @@ class ConfirmationPage extends React.Component {
...
-
+        <p>We may contact you if we have questions or need more information.
+        You can print this page for your records. <strong>If you need us to get
+        your private medical records from your doctor,</strong> you’ll need to
+        fill out an Authorization to Disclose Information to the VA (VA Form
+        21-4142). If you didn’t upload this form to your application, you can
+        download it here and mail it in.</p>
+        <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+        Download VA Form 21-4142</a>
+        <p>
```
Comment (cehsu) [inline src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: Do we want to use `.va-address-block` here (perhaps a question for @goldenmeanie or @peggygannon)? It would produce the following address format:
image

**Review by cehsu — APPROVED** [review]
LGTM. I had a few minor questions.

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 1 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: I don't think so. Looks like we usually use y/n as keys in other forms. I think booleans are reserved words so probably can't be used as keys in any case.

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 2 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: Probably, but I'm not sure you could use `hideIf` in combination with the standard `expandUnder` functionality, which has a lot of conveniences built in. Also, this approach is what we do for the 'which should I choose?' buttons.

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 3 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: 🤔 what's the advantage in doing so? Would that introduce a bunch of schema that we don't actually want in the UI for this page? I'm probably missing something about how forms work...

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx (hunk 4 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: That looks nice! Good find. Will wait for Alex / Peggy to confirm.

**Review by peggygannon — COMMENTED** [review]
@markgreenburg Please see my copy updates below. Let me know if you have any questions about my comments: 

**disability-benefits/526/apply-for-increase/supporting-evidence/0/private-medical-records-choice**
--Please add a period to the end of this sentence: 
No, my doctor has my medical records. 

**Expander text**
Please add a period to the end of this sentence: 
[Download VA Form 21-41-42].

**Which should I choose text**
Change: "We get your medical records" TO
Change: "We get your medical records from your doctor"
Update text to: 
We can get your medical records for you, but you'll first need to fill out an Authorization to Disclose Information to VA (VA Form 21-4142) so we can request your records. Getting your records might take us some time, and this could mean it'll take us longer to make a decision on your claim.<br>
[Download VA 21-4142].

**Supporting Evidence screen**
I recommend we remove this line: 
Or you can upload a completed VA Form 21-4142 to your online application. You'll have a chance later to upload your documents. 
 
**Confirmation page**
Please put this on its own line 
**If you need us to get your private medical records from your doctor,** you'll need to fill out an Authorization to Disclose Information to the VA (VA Form 21-4142). 

-Remove this sentence: 
If you didn't upload this form to your application, you can download it here and mail it in. 

- Add a period after this sentence: 
[Download VA Form 21-4142].

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx (hunk 4 above)
Comment (peggygannon) [reply src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: @markgreenburg Yes, the VA block address format looks good to me!

**Comment by markgreenburg** [comment]
@peggygannon updated screenshots with your copy edits. Please let me know if anything looks off!

**Comment by peggygannon** [comment]
@markgreenburg Hi, Mark. Looks good. 

One things that needs to be fixed: 
On the Supporting Evidence screen, there is an errant > in the zip code that should be deleted.

**Comment by markgreenburg** [comment]
Ah yikes, how did I miss that? Thanks for catching the zipcode weirdness, @peggygannon!

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx [hunk 5]
```diff
@@ -36,8 +36,25 @@ class ConfirmationPage extends React.Component {
       <div>
         <h3 className="confirmation-page-title">Claim submitted</h3>
         <p>We usually process claims within <strong>103 days</strong>.</p>
-        <p>We may contact you if we have questions or need more information. You can print this page for your records.</p>
-
+        <p>We may contact you if we have questions or need more information.
+        You can print this page for your records.</p>
+        <p>
+          <strong>If you need us to get your private medical records from your
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: @goldenmeanie It looks like we have access to `formData` in here, so we can show this section only if they chose the option that requires this. Should we show this information only if it's applicable?

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 3 above)
Comment (cvalarida) [reply src/applications/disability-benefits/526EZ/config/form.js]: Yeah, I don't think we want to do that. We need the `disabilities` property so we loop through it appropriately as an array page, but we shouldn't need the schema.

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 6]
```diff
@@ -539,6 +553,39 @@ const formConfig = {
...
+          }
+        },
+        privateMedicalRecordRelease: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-release',
+          showPagePerItem: true,
+          itemFilter: (item) => _.get('view:selected', item),
+          arrayPath: 'disabilities',
+          // TODO: Re-enable actual depends logic for page once 4142 PDF generation is working through vets-api
+          depends: () => false,
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/config/form.js]: Might make more sense to comment the whole page out so it doesn't get processed by the library at all. Minor optimization, but it's pretty simple, right?

File: src/applications/disability-benefits/526EZ/tests/config/privateRecordReleases.unit.spec.jsx [hunk 7]
```diff
@@ -7,7 +7,12 @@ import { DefinitionTester, fillData } from '../../../../../platform/testing/unit
 import formConfig from '../../config/form.js';
 import initialData from '../schema/initialData.js';
 
-describe('Disability benefits 526EZ VA facility', () => {
+// We don't allow users to hit this page for the time being, because we have
+// nothing that we can do with this information. The page is always skipped in
+// the config. Also, these tests possibly duplicate
+// privateMedicalRecordsRelease.unit.spec.jsx so we probably want to delete one
+// of these two files down the road.
+xdescribe('Disability benefits 526EZ VA facility', () => {
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/tests/config/privateRecordReleases.unit.spec.jsx]: Uh...yeah.... :thinking:

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx (hunk 5 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: Oh, that's an interesting thought. Would probably make things less confusing.

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 6 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: Yeah, having that much commented out seemed sloppy, but assuming this is just a temporary situation, you're probably right. Plus, I could get rid of the extra `depends` code.

Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: Actually, commenting out the whole page breaks the test files, and also requires commenting out some imports to satisfy es-lint. I think it's probably an easier change to just keep the extra `depends`, as-is.

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx (hunk 5 above)
Comment (goldenmeanie) [reply src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: Are you talking about line 42 in particular? Or the whole thing?

Comment (goldenmeanie) [reply src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: I think the message that we will contact you if necessary is generally true. But the private medical records chunk could be conditional.
